conclusions/rq1.md

```diff
@@ -0,0 +1,6 @@
+## What can be learned from existing (non-software) communities?
+Looking at the different communities, it can be concluded that most of the communities have one thing in common: they work because they have a goal. In the case of the intentional communities: all but the practical communities have their ideals as a goal to fulfill. The practical communities lack a goal, but those communities work because they fulfill the goal of making life easier.  
```

Comment:
`The practical communities lack a goal, but those communities work because they fulfill the goal of making life easier.`

I would rephrase this to be something like: `The practical communities make life easier rather than fulfilling an explicit goal.`